src/data/faq_de.json

```diff
@@ -1,2833 +1,2907 @@
-                        "anchor": "hearing_impaired",
-                        "active": false,
-                        "textblock": [
-                            "Grundsätzlich: Die App ist barrierefrei entwickelt worden (siehe <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/translations/scoping_document.de.md#barrierefreiheit' target='_blank' rel='noopener noreferrer'>Scoping-Dokument</a>) und unterstützt die gängigen Bedienungshilfen des Smartphone-Betriebssystems.",
-                            "In der Regel ist eine Kommunikation per Telefon nicht notwendig. Im Falle eines positiven Befunds erhalten Sie das Testergebnis standardmäßig über einen PCR-Test-QR-Code, den Sie mit der App einscannen. Wenn das Testzentrum bzw. das Labor diesen Weg nicht unterstützt oder Sie den QR-Code nicht scannen möchten, erhalten Sie eine TAN. Nur wenn Sie mit einem positiven Befund weder einen QR-Code noch eine TAN erhalten, müssten Sie die TAN-Hotline anrufen, was nicht direkt aus der App möglich ist.",
-                            "Sollten Sie wegen einer Hörschwäche nicht telefonieren können, empfiehlt es sich, darauf gleich beim Test hinzuweisen, da auch hier für die weitere Kommunikation besondere Maßnahmen einzuhalten sind. So kann sichergestellt werden, dass Sie die TAN - wenn benötigt - gleich schriftlich erhalten."
-                        ]
-                    },
-                    {
-                        "title": "Wie oft wird der Risiko-Status aktualisiert?",
-                        "anchor": "risk_update_24",
-                        "active": false,
-                        "textblock": [
-                            "Die Corona-Warn-App aktualisiert den Risiko-Status im Abstand von mindestens 24 Stunden. Dabei kann der genaue Zeitpunkt auch einige Stunden nach dem Zeitpunkt der Aktualisierung am Vortag liegen.",
-                            "Seit Version 1.7 der Corona-Warn-App wird der Risikostatus bis zu 6 mal täglich aktualisiert, wenn eine WLAN-Verbindung vorhanden ist. Die wiederholte Abfrage wird automatisch durchgeführt. Bei einer mobilen Internetverbindung wird der Risikostatus aufgrund der <a href='#mobile_data_costs' target='_blank' >Zero-Rating-Verpflichtungen</a> der Provider lediglich einmal täglich abgefragt."
-                        ]
-                    },
-                    {
-                        "title": "Wenn ich die App außerhalb eines WLANs nutze, kommen dann Kosten für die Übertragung mobiler Daten auf mich zu?",
-                        "anchor": "mobile_data_costs",
-                        "active": false,
-                        "textblock": [
-                            "Die deutschen Netzbetreiber haben sich zu einem sog. Zero-Rating verpflichtet. Das bedeutet, dass für die Nutzung der Corona-Warn-App das mobile Datenvolumen sowohl in Deutschland als auch innerhalb des EU-Auslands kostenfrei zur Verfügung gestellt wird. Die mobilen Daten, welche
```

Comment:
"Covid-19" => "COVID-19"

Reply:
@MikeMcC399 It seems like you are reviewing deletions (GitHub has a red background for them) and all your reviews show "Outdated". No idea why, but just FYI.

Reply:
There must be something wrong here, this line is:

"Die deutschen Netzbetreiber haben sich zu einem sog. Zero-Rating verpflichtet. Das bedeutet, dass für die Nutzung der Corona-Warn-App das mobile Datenvolumen sowohl in Deutschland als auch innerhalb des EU-Auslands kostenfrei zur Verfügung gestellt wird. Die mobilen Daten, welche die Corona-Warn-App überträgt, um Diagnoseschlüssel ('Positivkennungen') hochzuladen oder im Fall einer Risiko-Begegnung vom Backend-Server benachrichtigt zu werden, kosten Sie keinen Cent und werden nicht von Ihrem vertraglichen Datenvolumen abgezogen."

There is no "Covid-19" in this line, or am I overlooking something?